guides/v2.1/cloud/cdn/configure-fastly.md

```diff
@@ -129,6 +138,7 @@ You cannot use the Fastly service in Integration environments.
 		git add -A; git commit -m "Install Fastly"; git push origin <branch name>
```

Comment:
This is not part of your PR, but just pointing out that we need to have a consistent way to perform multiple git commands, since people like to copy the commands from the docs. The format we have been using is `git add -A && git commit -m "Note about update" && git push origin <branch name>`